### ISSUE TITLE
A Compatibility of G-helper with ARC Browsers

### ISSUE THREAD

**Issue by bammcool2011** [seerge/g-helper#2573]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

Dear Seerge,
I have an issue with G-helper when using _ARC Browser_ as the default software in Windows 11. Whenever I click on Version 0.1xx.x or Driver links in the updates menu, ARC does not open. However, when I change to other browsers, the external links work normally.

After closing G-helper and restarting, the external links started working and the ARC opened up. Then I restarted my PC and the issue reoccurred.

### How to reproduce the bug?

1. Click on Version 0.1xx.x or Tried to download drivers
2. ARC not opened
3. Restart a program
4. External Links work normally, ARC opened
5. Restart PC
6. The issue come up again

### Logs

[log.txt](https://github.com/seerge/g-helper/files/15278602/log.txt)

### Device and Model

ASUS TUF Gaming A15 FA507NU

### Armoury Crate

Never installed

### Asus Services

None

### Version

0.170.0

### OS

Windows 11 version 23H2

**Comment by seerge** [comment]
@bammcool2011 hello.

G-Helper doesn't launch browser directly. It just tells windows to open URL with whatever method it has for that. Which normally would trigger browser :) 

It looks more like an issue with your ARC Browser by itself cause it doesn't associate itself with links in windows correctly.

Are you able to launch this browser at least from any other application in windows?

**Comment by seerge** [comment]
@bammcool2011 P.S. I have installed this "ARC browser" .. and it actually opens absolutely fine from all links in G-Helper
![Screenshot 2024-05-10 212711](https://github.com/seerge/g-helper/assets/5920850/76539135-5cec-44d9-a310-d5d34710c0fb)

I just allowed all default associations it has requested in windows

So it looks like problem is just somewhere in windows settings on your specific device.

I'm closing this as this is not G-Helper issue anyway

Thanks.

**Comment by bammcool2011** [comment]
Thank you for a quick response. 
I'm sorry for not mentioning earlier that I also have G-helper set to run during Windows start-up. It appears that when I set ARC as the default program and allowed G-helper to run with Windows after restarting or starting the computer again, the issue occurred. Some of my friends are also facing similar issues too. 

Could you please look into this issue again? If it's not related to G-helper, I will escalate this problem to the ARC team later.

**Comment by seerge** [comment]
@bammcool2011 I have re-checked it again, with G-Helper running on startup - it still opens links fine.

But what I have noticed is that if G-Helper is launched as admin , ARC just refuses to launch (cause link is opened with same permissions). 

Cause this ARC just doesn't launch as admin at all (even if you click on it's icon -> select run as admin)

So it's probably some issue there :) I can't fix it

I have also noticed that ARC keep forgetting my login and asking me to login every time i start it. So it looks like that's not the only problem it has so far :))
